src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -1612,7 +1630,8 @@
         {
           "condition": "(useCsharpMarkup)",
           "exclude": [
-            "MyExtensionsApp.1/**/*.xaml"
+            "MyExtensionsApp.1/**/*.xaml",
+            "MyExtensionsApp.1/Styles/.gitignore"
```

Comment:
Can it also be conditional to material?

Reply:
When we're not using material the entire Styles directory is ignored